cmd/syncer/main.go

```diff
@@ -294,6 +294,8 @@ func initSyncerComponents(ctx context.Context, clusterFlavor cnstypes.CnsCluster
 					"openshift": "Openshift",
 					"wcp":       "Supervisor",
 					"vmware":    "TanzuKubernetesCluster",
+					"eks":       "EKS",
```

Comment:
No in this PR, but can we break this down into method so we can write some unit tests for this logic.

Reply:
Will address this in another PR